backend/FacebookEventsFeed/index.js

```diff
@@ -0,0 +1,80 @@
+const fs = require('fs');
+const https = require('https');
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3({region: 'eu-central-1'});
+const path = require('path');
+
+
+require('dotenv').config({path: path.join(__dirname, 'deploy.env')})
+exports.handler = (event, context) => {
+  getEventsFeed().then(res => {
+    if(res.error) {
```

Comment:
please follow javascript standards `if (res.error) {` instead of `if(res.error) {`